src/common/stores/advertisementStore.ts

```diff
@@ -0,0 +1,14 @@
+import { ref, computed } from 'vue'
+import { defineStore } from 'pinia'
+import { User } from '../models/advertisementModels'
+
+export const useUserStore = defineStore('user', () => {
+  const currentAdvertisement = ref({})
+  const isLoggedIn = computed(() => currentAdvertisement.value instanceof Advertisement)
```

Comment:
what does this even mean in the context of an advertisement?

Reply:
nothing, started working on advertisements but did not finish